src/EventSubscriber/CacheableResponseSubscriber.php

```diff
@@ -21,14 +22,42 @@ class CacheableResponseSubscriber implements EventSubscriberInterface {
    */
   protected $logger;
 
+  /**
+   * Configuration Factory.
+   *
+   * @var \Drupal\Core\Config\ConfigFactory
+   */
+  protected $configFactory;
+
   /**
    * Constructs a new DefaultExceptionHtmlSubscriber.
    *
    * @param \Psr\Log\LoggerInterface $logger
    *   The logger service.
+   * @param \Drupal\Core\Config\ConfigFactory $configFactory
+   *   Configuration for this module.
    */
-  public function __construct(LoggerInterface $logger) {
+  public function __construct(LoggerInterface $logger, ConfigFactory $configFactory) {
     $this->logger = $logger;
+    $this->configFactory = $configFactory;
+  }
+
+  /**
+   * Returns whether entity_list tags should be overridden.
+   *
+   * Overriding these tags was the initial behavior of the 1.0 version of this
+   * module. That is no longer recommended.
+   */
+  public function getOverrideListTagsSetting() {
+    $config = $this->configFactory->get('pantheon_advanced_page_cache.settings');
+    // Only return FALSE if this config value is really set to false.
+    // A null value should return TRUE for backwards compatibility.
+    if ($config->get('override_list_tags') === FALSE) {
+      return FALSE;
+    }
+    else {
+      return TRUE;
+    }
```

Comment:
This is simpler to read, imo.

```suggestion
    return TRUE;
```